src/System.ServiceModel.Primitives/pkg/System.ServiceModel.Primitives.pkgproj

```diff
@@ -28,6 +31,14 @@
       <!-- Supported inbox by all ns1.0-1.2 frameworks -->
       <Value>.NETStandard,Version=v1.0;.NETStandard,Version=v1.1;.NETStandard,Version=v1.2</Value>
     </ValidatePackageSuppression>
+    <!-- Impl shim for System.ServiceModel.dll -->
```

Comment:
In this case, should we put the shim in netstandard2.0 folder under lib in the package?